src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-count/jj-count.component.ts

```diff
@@ -83,11 +126,63 @@ export class JJCountComponent implements OnInit {
         this.jjDisputedCount.lesserOrGreaterAmount = this.form.get('lesserOrGreaterAmount').value;
         this.jjDisputedCount.totalFineAmount = this.form.get('totalFineAmount').value;
         this.jjDisputedCount.includesSurcharge = this.inclSurcharge == "yes" ? this.IncludesSurcharge.Y : this.IncludesSurcharge.N;
+        if (this.jjDisputeInfo.hearingType === this.HearingType.CourtAppearance) {
+          this.jjDisputedCount.jjDisputedCountRoP.finding = this.form.get('jjDisputedCountRoP').get('finding').value;
```

Comment:
can you remove the duplicated code. Extract `this.form.get('jjDisputedCountRoP')` to a variable at the top. Will make more readable and increase minification effectiveness.